target/linux/ath79/dts/qca9531_glinet_gl-e750.dts

```diff
@@ -45,6 +45,13 @@
 
 &pcie0 {
 	status = "okay";
+
+	wifi@0,0 {
+		compatible = "qcom,ath10k";
+		reg = <0x0000 0 0 0 0>;
+		nvmem-cells = <&macaddr_art_0 2>;
+		nvmem-cell-names = "mac-address";
```

Comment:
It seems that the PCIe ath10k wifi never work on this device due to missing calibration eeprom.
https://forum.gl-inet.com/t/e750-5ghz-wifi-not-working-properly-stock-openwrt-23-05-2/35404/4

https://github.com/openwrt/openwrt/pull/15544

For this PR, LGTM.